### PR TITLE
Update expo-cli docs

### PR DESCRIPTION
**Pull request by AdamJNavarro**
Purpose: Make the expo-cli docs a bit more beginner-friendly by painting a clearer picture of what it can be used for.

Changes:
- Create a bulleted list of the main features
- In-lined the installation steps so users don't have to navigate away from the page

Will apply changes to remaining versions once agreed upon.

# Why

Please describe the motivation for this PR, and link to relevant GitHub issues, forums posts, or feature requests.

# How

How did you build this feature or fix this bug and why?

# Test Plan

Please describe how you tested this change and how a reviewer could reproduce your test, especially if this PR does not include automated tests! If possible, please also provide terminal output and/or screenshots demonstrating your test/reproduction.

### CODE REVIEW

**Review by cruzach — COMMENTED** [review]
Looks good to me! Only thoughts I have would be to change “enhances your workflow” to something more concrete like “makes development faster and easier” just bc people reading these docs are most likely beginners so I think the more obvious what it does is, the better. Also for the same reason, linking to the publishing/building docs pages in those specific bullet points, so that they can easily find what we mean when we say something like “Publishing your JS and assets,” which is now second-nature to us but isn’t if you’re new to Expo.

File: docs/pages/versions/unversioned/workflow/expo-cli.md [hunk 1]
```diff
@@ -2,11 +2,21 @@
 title: Expo CLI
 ---
 
-Expo CLI is a command line interface for developing Expo apps. It also includes a web-based inteface (Expo Dev Tools) for using some of the most often used features also from a quick to use graphical interface.
+Expo CLI is a command line interface that enhances your workflow by empowering developers to perform actions such as:
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/workflow/expo-cli.md]:
```suggestion
Expo CLI is a command line app that is the main interface between a developer and Expo tools. You'll use it for a variety of tasks, such as:
```

File: docs/pages/versions/unversioned/workflow/expo-cli.md [hunk 2]
```diff
@@ -2,11 +2,21 @@
...
+
+- Creating a new Expo project
+- Serving your app so you can begin development
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/workflow/expo-cli.md]:
```suggestion
- Developing your app: running the project server, viewing logs, opening your app in a simulator 
```

File: docs/pages/versions/unversioned/workflow/expo-cli.md [hunk 3]
```diff
@@ -2,11 +2,21 @@
 title: Expo CLI
 ---
 
-Expo CLI is a command line interface for developing Expo apps. It also includes a web-based inteface (Expo Dev Tools) for using some of the most often used features also from a quick to use graphical interface.
+Expo CLI is a command line interface that enhances your workflow by empowering developers to perform actions such as:
+
+- Creating a new Expo project
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/workflow/expo-cli.md]:
```suggestion
- Creating new projects
```

File: docs/pages/versions/unversioned/workflow/expo-cli.md [hunk 4]
```diff
@@ -2,11 +2,21 @@
...
+- Serving your app so you can begin development
+- Publishing your JS and assets
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/workflow/expo-cli.md]:
```suggestion
- Publishing your app JavaScript and other assets and managing releasing them over the air
```

File: docs/pages/versions/unversioned/workflow/expo-cli.md [hunk 5]
```diff
@@ -2,11 +2,21 @@
...
+- Building `apk` and `ipa` files to be uploaded to the App Store and Play Store
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/workflow/expo-cli.md]:
```suggestion
- Building binaries (`apk` and `ipa` files) to be uploaded to the App Store and Play Store
```

File: docs/pages/versions/unversioned/workflow/expo-cli.md [hunk 6]
```diff
@@ -2,11 +2,21 @@
...
+- Uploading Apple Credentials and Google Keystores
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/workflow/expo-cli.md]:
```suggestion
- Managing Apple Credentials and Google Keystores
```

File: docs/pages/versions/unversioned/workflow/expo-cli.md [hunk 7]
```diff
@@ -2,11 +2,21 @@
...
+
+It also includes a web-based interface, named Expo Dev Tools, which enables you to use some of the most often used features from a quick-to-use graphical interface. We’ve only scratched the surface of what expo-cli can do so far. Be sure to check out all the possible commands below!
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/workflow/expo-cli.md]: You may use the CLI in your terminal or use the web based interface (it opens automatically by default, or you can press `d` from the CLI to open it on demand). The web interface enables you to use some of the most often used features from a quick-to-use graphical interface. We’ve only scratched the surface of what expo-cli can do so far. Be sure to check out all the possible commands below!

**Review by brentvatne — CHANGES_REQUESTED** [review]

**Pull request merged**